docs/docs/tracing/sdk/search.mdx

```diff
@@ -0,0 +1,259 @@
+---
+sidebar_position: 2
+sidebar_label: Query Traces
+---
+
+import { APILink } from "@site/src/components/APILink";
+import { Card, CardGroup, SmallLogoCard } from "@site/src/components/Card";
+import TOCInline from "@theme/TOCInline";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+
+# Query Traces
+
+This page describes how to query traces logged to MLflow.
+
+
+## Client vs Fluent API
```

Comment:
Standard vs Client APIs? If we're removing the fluent term from the main page, might as well keep it consistent.

Reply:
Yea that makes sense. Not sure if pandas one is "standard"; maybe "high-level" is slightly more accurate.